api/src/main/java/org/pmiops/workbench/db/model/DbInstitutionEmailAddress.java

```diff
@@ -0,0 +1,71 @@
+package org.pmiops.workbench.db.model;
+
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "institution_email_address")
+public class DbInstitutionEmailAddress {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @Column(name = "institution_email_address_id")
+  private long institutionEmailAddressId;
+
+  @ManyToOne(fetch = FetchType.LAZY)
+  @JoinColumn(name = "institution_id")
+  private DbInstitution institution;
+
+  @Column(name = "email_address", nullable = false)
+  private String emailAddress;
+
+  public DbInstitutionEmailAddress() {}
+
+  public DbInstitutionEmailAddress(DbInstitution institution, String emailAddress) {
+    this.institution = institution;
+    this.emailAddress = emailAddress;
+  }
+
+  public DbInstitution getInstitution() {
+    return institution;
+  }
+
+  public String getEmailAddress() {
+    return emailAddress;
+  }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (!(o instanceof DbInstitutionEmailAddress)) {
+      return false;
+    }
+
+    DbInstitutionEmailAddress that = (DbInstitutionEmailAddress) o;
+
+    if (institutionEmailAddressId != that.institutionEmailAddressId) {
```

Comment:
Sorry.. i was meaning that you could remove these 2 if statements and replace with the return suggestion

Reply:
So basically, you would replace
`if (institutionEmailAddressId != that.institutionEmailAddressId) {`
`return false;`
`}`
`if (!institution.equals(that.institution)) {`
`return false;`
`}`
`return emailAddress.equals(that.emailAddress);`

with 

`return institutionEmailAddressId == that.institutionEmailAddressId`
`        && institution.equals(that.institution)`
`        && emailAddress.equals(that.emailAddress);`

Reply:
Yes of course.  Merged these on autopilot.  Fixed now.